miniapps/meshing/mesh-explorer.cpp

```diff
@@ -496,7 +497,37 @@ int main (int argc, char *argv[])
 
       if (mk == 't')
       {
-         mesh->Transform(transformation);
+         int type;
+         cout << "Choose a transformation:\n"
+              "0: User-defined transform through mesh-explorer::transformation()\n"
+              "1: Kershaw transform ---> " << flush;
```

Comment:
I would prefer to see a newline before the `--->`. This would be more similar to the main menu and the refinement choices etc.

Reply:
Actually if you really want to follow the current pattern you should use letters rather than numbers and ')' rather than ":". Something like:
```
u) User-defined transform through mesh-explorer::transformation()
k) Kershaw transform
 --> 
```

Reply:
Good idea, done!